Dockerfile

```diff
@@ -2,22 +2,18 @@ FROM python:3.12-slim as application
 
 WORKDIR /app
 
-# arm architecture platform builds need postgres drivers installing via apt
-ARG TARGETARCH
-RUN if [ "${TARGETARCH}" != "amd64" ]; then apt-get update && apt-get install -y gcc libpq-dev && rm -rf /var/lib/apt/lists/*; fi;
```

Comment:
I'm not sure why this was removed?

Reply:
We didn't need those in the first place.

Reply:
Fair point... I just bothered to actually read the comment.